src/tensor_operations/matrix_decomposition.jl

```diff
@@ -159,6 +159,7 @@ function svd(
     AC = permute(AC, cL, cR)
   end
 
```

Comment:
The code in this file was accidentally included. I will remove it and close this comment after I do.

Reply:
This has all been removed.